chronosQ-core/src/main/java/flipkart/cp/convert/chronosQ/core/SchedulerData.java

```diff
@@ -0,0 +1,27 @@
+package flipkart.cp.convert.chronosQ.core;
+
+import com.google.common.base.Strings;
+import lombok.AllArgsConstructor;
+
+import java.util.Optional;
+
+@AllArgsConstructor
+public class SchedulerData {
+    private final String key;
+    private String value;
+
+    public String getKey() {
+        return key;
+    }
+
+    public Optional<String> getValue() {
```

Comment:
Can this be really optional ? Since when pushing to sink we publish `getValue().orElse("")` which doesn't make sense as sink won't be able to process anything with a empty string value.

Reply:
Changed as discussed.